client/app/bundles/course/statistics/handles.ts

```diff
@@ -16,12 +14,8 @@ export const videoWatchHistoryHandle: DataHandle = (match) => {
       const { data } = await CourseAPI.users.fetch(userId);
 
       return {
-        activePath: `/courses/${courseId}/statistics`,
+        activePath: `/courses/${courseId}/statistics/students`,
         content: [
-          {
-            title: StatisticsIndex.handle,
```

Comment:
Why is this removed?

Reply:
this line seems like serving no purpose at all to the component here (this one is videoWatchHistoryHandle, which basically has no relation at all with the StatisticsIndex).

Also, removing this does not impact the related page as well (`video_submissions`), therefore I am wondering why this line should be here in the first place.

Reply:
Do you know what this is even for?

Reply:
understand that this is for breadcrumbs, apologies for the oversight.
Have been reverted and fixed accordingly following the new conventions for course statistics routing